### ISSUE TITLE
Should this project continue?

### ISSUE THREAD

**Issue by jamespic** [jamespic/SpaceSuit#21]
MetaMask have recently integrated Ledger Nano S support (https://github.com/MetaMask/metamask-extension/pull/5050). As such, it's unclear whether SpaceSuit still solves a problem the community has.

The approach MetaMask takes is one that sits slightly uneasily with me (it depends on an iframe pointing at an external site), but I don't know whether this is important enough to justify the time it'd take to keep SpaceSuit maintained. 

I've got 0.3.0 about ready for release (I'm just refactoring some tests), which fixes several issues, so releasing that wouldn't be a too much effort.

**Comment by lazaridiscom** [comment]
Me a bit "uneased", too - although I suggested the IFRAME/Messaging solution as the most favorable path (see MetaMask/metamask-extension/issues/4060).

I would keep your spacesuit project running, kind of an "alternative solution path" thing.

The top-most favorable solution-path is still blocked, thus I suggested to keep the related issue open, and to push the WebUSB implementation:

MetaMask/metamask-extension/issues/4205

**Comment by Sentrycs** [comment]
@jamespic Are you still involved with this project?

**Comment by jamespic** [comment]
@Sentrycs I haven't done much development on it lately. I don't know of any bugs that need fixing, so new development would be new features and better documentation. Which I haven't found time to do.

**Comment by Sentrycs** [comment]
@jamespic Fair enough. We want to fork Metamask into using our own Security SaaS. Would you mind to share your experience with SpaceSuit? We might have a couple of questions along the road.

**Comment by jamespic** [comment]
@Sentrycs Yep, that's no problem. Rather than spam anyone watching this issue, we can continue the discussion over email. I'm james_pic@hotmail.com.
